spec.html

```diff
@@ -16175,16 +16175,16 @@
               for (let key of Reflect.ownKeys(obj)) {
                 if (typeof key === "string") {
                   let desc = Reflect.getOwnPropertyDescriptor(obj, key);
-                  if (desc && !visited.has(key)) {
+                  if (desc &amp;&amp; !visited.has(key)) {
```

Comment:
@mathiasbynens, thanks for the article. I dropped this commit and will make another PR to unescape all unambiguous ampersands in `spec.html`.